base/strings/string.jl

```diff
@@ -425,6 +425,17 @@ function reverse(s::String)
     String(buf)
 end
 
+"""
+    repeat(s::String, r::Integer)
+
+Repeat a string `r` times.
```

Comment:
@tkelman thanks will update. This means I should include:
```julia
Base.repeat(::String, r::Integer)
Base.repeat(::Char, r::Integer)
```

Actually should we not add the following fallback method ?
```julia
repeat(s::AbstractString, r::Integer) = repeat(string(s), r)
```